pkgs/applications/science/biology/MACS2/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, python3, fetchurl }:
+
+python3.pkgs.buildPythonPackage rec {
+    pname = "MACS2";
+    version = "2.2.7.1";
+
+    src = python3.pkgs.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "1rcxj943kgzs746f5jrb72x1cp4v50rk3qmad0m99a02vndscb5d";
+    };
+
+    propagatedBuildInputs = with python3.pkgs; [ numpy ];
+    doCheck = false;
```

Comment:
Please add a comment why you disabled tests.

Reply:
Done